### PR TITLE
Slightly more generic dialog styles

### PR DESCRIPTION
**Pull request by keithpitt**
- [x] Removed forced padding and `center`'ness
- [x] Width can now be configured

### CODE REVIEW

File: app/components/agent/Index/installation.js [hunk 1]
```diff
@@ -67,12 +67,14 @@ class AgentInstallation extends React.Component {
         </Panel.Header>
         <Panel.Section>
           See the <a className="blue hover-navy text-decoration-none hover-underline" href="/docs/agent/installation">agent installation documentation</a> for the full details of installing and configuring your Buildkite agents for any machine or architecture.
-          <Dialog closeable={false} isOpen={this.state.isDialogOpen}>
-            <p className="mt0 h2">
-              <Emojify text=":tada:" /> You’ve successfully connected {agentName}!<br/>
-              You’re all ready to run builds!
-            </p>
-            <Button link={`/${organization.slug}`}>Take me to the dashboard</Button>
+          <Dialog closeable={false} isOpen={true}>
```
Comment (ticky) [inline app/components/agent/Index/installation.js]: Why is this forced open?

File: app/components/shared/Dialog.js [hunk 2]
```diff
@@ -154,7 +154,7 @@ class Dialog extends React.Component {
     }
 
     return (
-      <DialogBox>
+      <DialogBox width={this.props.width || 500}>
```
Comment (ticky) [inline app/components/shared/Dialog.js]: This should probably be a `defaultProps` thing

File: app/components/agent/Index/installation.js (hunk 1 above)
Comment (keithpitt) [inline app/components/agent/Index/installation.js]: Oh lol, typo.

**Comment by keithpitt** [comment]
@ticky made those changes!

**Comment by keithpitt** [comment]
Except the build hates me

**Pull request merged**